alphalens/plotting.py

```diff
@@ -709,7 +721,15 @@ def plot_cumulative_returns_by_quantile(quantile_returns, ax=None):
         f, ax = plt.subplots(1, 1, figsize=(18, 6))
 
     ret_wide = quantile_returns.reset_index()\
-        .pivot(index='date', columns='quantile', values=1)
+        .pivot(index='date', columns='quantile', values=period)
+
+    if period > 1:
+        daily_returns = lambda ret, period: (
+            np.nanmean(ret) ** (1. / period)) - 1
+        period_ret_wide = ret_wide.add(1)**period
+        ret_wide = period_ret_wide.rolling(
```

Comment:
Yeah, it's a pandas 18 (18.1?) feature. I wasn't sure what was the target pandas version. I will use the old method then (and eventually we have to switch back to this version as the new one complains about deprecated pandas.rolling_apply :) )